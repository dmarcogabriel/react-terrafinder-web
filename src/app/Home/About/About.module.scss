@import 'styles';

.about {
  padding: 10rem 2rem 3rem;

  > h1 {
    font-size: 2.5rem;
    text-align: center;
    margin: 1rem 0;
  }

  .card {
    border-radius: 0.25rem;
    box-shadow: $shadow-base;
    padding: 1rem;

    .row {
      display: flex;
      flex-direction: column;

      > div > img {
        margin: 1rem 0;
        width: 100%;

        @media ($desktop) {
          width: 60%;
        }
      }

      @media ($desktop) {
        flex-direction: row;

        > div {
          width: 50%;
          padding: 2rem 2rem;
          display: flex;
          flex-direction: column;
          align-items: center;
          justify-content: center;

          > p {
            line-height: 2rem;
          }
        }
      }
    }

    .aboutValues {
      border-top: 1px solid $orange;
      margin: 0 2rem;
      padding: 1rem 0;

      > h2 {
        color: $primary-dark;
        font-size: 2rem;
        font-weight: normal;
        text-align: center;
        margin: 0.5rem 0;
      }

      > p {
        text-align: center;
      }

      > ul > li {
        text-align: center;
        line-height: 1.75rem;
      }
    }
  }
}
