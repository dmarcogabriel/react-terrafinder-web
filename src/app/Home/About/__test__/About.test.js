import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
// import { render } from '@testing-library/react';
import About from '..';

describe('<About />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<About />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  // Add tests here...

  it('matches snapshot', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
