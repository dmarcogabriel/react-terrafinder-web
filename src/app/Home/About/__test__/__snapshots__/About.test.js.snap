// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> matches snapshot 1`] = `
<div>
  <h1>
    Quem somos?
  </h1>
  <div>
    <div>
      <div>
        <p>
          O Brasil é o quinto país com maior extensão territorial do mundo, com cerca de 152,5 milhões de hectares (17,9% do território brasileiro) sendo terras e áreas agricultáveis. Segundo o Sistema Nacional de Cadastro Rural (SNCR), em 2013 o país possuía cerca de 5.6 milhões de imóveis rurais, ocupando 60% da área todal do Brasil.
        </p>
        <p>
          Analisando e estudando como é feito atualmente a compra e venda desses imóveis, a ideia da Terrafinder é facilitar esse processo que hoje é feito muitas vezes de maneira rudimentar e por meios como jornais, contatos pessoais, exploração e alguns canais digitais.
        </p>
        <p>
          Com uma onda de digitalização que hoje impacta até as gerações mais velhas, a ideia é construir um sistema que atenda as necessidades do público mais velho, mas que também dê liberdade para gerações mais novas poderem utilizar.
        </p>
        <p>
          A Terrafinder é uma startup que busca conectar e facilitar a vida de proprietários de imóveis rurais e de interessados em adquirir tais imóveis. Através de uma plataforma digital a ideia é permitir aos proprietários desses imóveis, anunciarem e promover suas vendas com facilidade e visibilidade para pessoas que tem interesse em comprar tais propriedades.
        </p>
      </div>
      <div>
        <img
          alt=""
          src="\\"Logo.svg\\""
        />
      </div>
    </div>
    <div>
      <h2>
        Missão
      </h2>
      <p>
        Conectar compradores e vendedores de imóveis rurais
      </p>
      <h2>
        Visão
      </h2>
      <p>
        Ser uma plataforma completa de soluções para o agrobusiness
      </p>
      <h2>
        Valores
      </h2>
      <ul>
        <li>
          Integridade
        </li>
        <li>
          Lealdade
        </li>
        <li>
          Compromisso
        </li>
        <li>
          Smart Work
        </li>
        <li>
          União e Equipe
        </li>
      </ul>
    </div>
  </div>
</div>
`;
